src/hermes_claude_agent_sdk/memory_skills.py

```
@@ -0,0 +1,214 @@
+    if value is None or isinstance(value, (bool, int, str)):
+        return value
+    if isinstance(value, float):
+        return value if math.isfinite(value) else {"__float__": str(value)}
+    if isinstance(value, Mapping):
+        pairs: list[tuple[str, Any]] = []
+        for key, item in value.items():
+            if not isinstance(key, str):
+                pairs.append((f"<key:{type(key).__name__}>", {"__type__": type(item).__name__}))
+                continue
+            pairs.append((key, _canonical_value(item)))
+        return {key: item for key, item in sorted(pairs)}
```

Ruff (0.16.2)

[warning] Unnecessary dict comprehension (rewrite using `dict()`)

Rewrite using `dict()`

(C416)